src/Evolution/Systems/GeneralizedHarmonic/GhDampingFunctions/GaussianPlusConstant.hpp

```diff
@@ -0,0 +1,123 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines GhDampingFunctions::GaussianPlusConstant.
+
+#pragma once
+
+#include <array>
+#include <pup.h>
+
+#include "Evolution/Systems/GeneralizedHarmonic/GhDampingFunctions/GhDampingFunction.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+/// \endcond
+
+namespace GhDampingFunctions {
+template <size_t VolumeDim, typename Fr>
```

Comment:
Sorry, it's not clear to me why you need this forward decl. Can you just remove it?